src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetStartMethod.cs

```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetStartMethod(object[] invokeMethodInp
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.Start(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Start(resourceGroupName, vmScaleSetName, instanceIds);
```

Comment:
breaking change?

Reply:
again, it looks like the method below is doign the actual work, so what is this change for?